src/database/database.py

```
@@ -60,7 +93,113 @@ def _initialize_database(self):
+            VALUES (?, ?, ?)
+        ''', (name, guild_id, channel_id))
+        conn.commit()
+        campaign_id = cursor.lastrowid
+        conn.close()
+        return campaign_id
+
+    def create_character(self, name, avatar_url, type_, player_id, campaign_id):
+        if isinstance(type_, CharacterType):
+            type_str = type_.value
+        else:
+            type_str = str(type_)
```

Ruff (0.8.2)

Use ternary operator `type_str = type_.value if isinstance(type_, CharacterType) else str(type_)` instead of `if`-`else`-block

Replace `if`-`else`-block with `type_str = type_.value if isinstance(type_, CharacterType) else str(type_)`

(SIM108)